src/components/Engines/Engines.test.tsx

```diff
@@ -1,71 +1,60 @@
 import React from 'react';
 import { mount } from 'enzyme';
 
+import { DetailContext } from '../../pages/Version';
+import { PackageMetaInterface } from '../../../types/packageMeta';
+
 import Engine from './Engines';
 
 jest.mock('./img/node.png', () => '');
 jest.mock('../Install/img/npm.svg', () => '');
 
-const mockPackageMeta: jest.Mock = jest.fn(() => ({
+const withEngineComponent = (packageMeta: PackageMetaInterface): JSX.Element => (
```

Comment:
I am not sure I like the name "withEngineComponent", because that is not really a HOC...it's just a func with a param..do you know what I mean?..

Reply:
Good point! I didn't notice the conceptual conflict with HOCs, I was being consistent with the suggestion from my other PR: https://github.com/verdaccio/ui/pull/150#discussion_r331141170

In this case I think the abstraction is a bit pointless since the tests and fixtures are so short, I've inlined the JSX into each test and it seems readable. This also avoids the hard naming problem 😅

Reply:
@bighuggies thank you!